Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -1,5 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 
+from Configuration.StandardSequences.Reconstruction_cff import *
```

Comment:
On 9/25/15 9:41 AM, Salavat Abdullin wrote:
> In Configuration/DataProcessing/python/RecoTLR.py
> <https://github.com/cms-sw/cmssw/pull/11481#discussion_r40436665>:
> 
>> @@ -1,5 +1,7 @@
>>  import FWCore.ParameterSet.Config as cms
>>  
>> +from Configuration.StandardSequences.Reconstruction_cff import *
> 
> Hongxuan Liu has suggested to use directly ESProducer (of those
> SeverityLevel setings), so
> 
> I presume something like this should be OK ?
> from RecoLocalCalo.HcalRecAlgos.hcalRecAlgoESProd_cfi import *

possibly,
but try to avoid "import *" and keep the import inside the customize
function definition.


> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/11481/files#r40436665>.
>